test/bigtest/helpers/PluginHarness.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import noop from 'lodash/noop';
+
+import { Pluggable } from '@folio/stripes/core';
+
+const PluginHarness = props => (
+  <Pluggable
+    aria-haspopup="true"
+    type="find-po-import-profile"
+    id="clickable-find-import-profile"
+    searchLabel="Choose Profile"
+    marginTop0
+    searchButtonStyle="link"
+    addLines={noop}
+    {...props}
+  >
+    <span data-test-no-plugin-available>No plugin available!</span>
```

Comment:
Please use translation string here and in other places

Reply:
Done